src/components/WalletSelect/WalletSelect.js

```diff
@@ -426,7 +426,7 @@ const WalletSelect = (props) => {
                               <Row>
                                 <Col xs="6" className="mt-1">
                                   <ShareLink
-                                    url={asset.tokenAddress}
+                                    url={asset.poolAddress}
```

Comment:
Fixed addresses copied and handed to metamask 'watch' function throughout walletSelect